src/components/playback/playbackmanager.js

```diff
@@ -1640,6 +1640,7 @@ class PlaybackManager {
             }
 
             changeStream(player, ticks);
+            Events.trigger(player, 'seek', [ticks]);
```

Comment:
Ideally, we should receive this event after the seeking is complete.
I believe that `timeupdate` is the correct event.